clang/docs/ReleaseNotes.rst

```diff
@@ -110,6 +110,11 @@ C++20 Feature Support
   templates (`P1814R0 <https://wg21.link/p1814r0>`_).
   (#GH54051).
 
+- __cpp_concepts macro now updated to `202002L`, as we are confident that,
+  modulo a handful of bugs and core issues, that our concepts implementation is
```

Comment:
We have fairly large hands https://github.com/llvm/llvm-project/labels/concepts - but i agree doing that early in the 19 cycle makes sense